azure-pipelines-v3.yml

```diff
@@ -75,25 +75,15 @@ jobs:
       DOCS_OPS_TOKEN: $(DOCS_OPS_TOKEN)
       CODECOV_TOKEN: $(CODECOV_TOKEN)
 
-  # Push to sandbox MyGet feed
+  # Push to sandbox azure devops feed
   - task: NuGetCommand@2
     condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/v3'))
     displayName: Push to Sandbox MyGet Feed
     inputs:
       command: push
       packagesToPush: drop/**/*.nupkg
-      nuGetFeedType: external
```

Comment:
I thought we should push to other two feeds - would you take a look at the description in the item?

Reply:
Please check my comment in the work item